User Guide.md

```diff
@@ -274,54 +283,66 @@ bash $ <ab.json jtc -tc
 { "size": 56 }
-bash $ 
+bash $
 ```
-if size only required (i.e., w/o printing the input JSON), then use `-zz` option:
+To print the total without printing the input JSON, use `-zz`:
+
 ```bash
 bash $ <ab.json jtc -zz
 56
-bash $ 
+bash $
 ```
 
-
 ### Validating JSON
-When JSON is read (from a file, or from `stdin`), it get parsed and validated. If an invalid JSON is detected, a short exception
-message will be displayed, e.g,:
-```bash
-bash $ <ab.json jtc
+
+When JSON is read (from a file or `stdin`), it gets parsed and validated. If
+invalid JSON is detected, a short exception message will be displayed:
+```bash bash $ <ab.json jtc
 jtc json parsing exception (<stdin>:1214): unexpected_end_of_line
-bash $ 
+bash $
 ```
-and though the message lets us knowing that there's a problem with the input JSON, it not very informative with regards whereabouts the
-the problem. To visualize the spot where the problem is, as well as its locus pass a single debug option (`-d`):
+
+The error messsage doesn't explain where the problem is, the `-d` flag (single
+debug) will help:
+
 ```bash
 bash $ <ab.json jtc -d
 .display_opts(), option set[0]: -d (internally imposed: )
 .init_inputs(), reading json from <stdin>
 .exception_locus_(), ...e": 80206,|            "state": "CO,|            "street address": "6213...
 .exception_spot_(), --------------------------------------->| (offset: 1214)
 jtc json parsing exception (<stdin>:1214): unexpected_end_of_line
-bash $ 
+bash $
 ```
-the vertical pipe symbol `|` in the debug showing JSON locus replaces new lines, thus it becomes easy to spot the problem. 
-The offset (`1214` in the example) is given in _unicode UTF-8_ characters from the beginning of the input/file/stream.
-In that particular failure instance, `jtc` found the end of a line, while _JSON string_ `"Co,` is still open (JSON standard does not
-permit multi-line strings). To fix that, the missing quotation mark to be added
+
+The vertical pipe symbol `|` in the debug output shows the location
+(`exception_locus .. offset:1214`) of the problem.
+
+The offset (`1214` in the example) is given in _unicode UTF-8_ characters from
+the beginning of the input/file/stream.  In this example `jtc` found an
+unexpected end of line in `"CO,` (the JSON standard doesn't permit multi-line
+strings). To fix this, the missing quotation mark must be added.
+
+Multiple debug flags (`-dd`, `-ddd`) can be used to gain greater insight.
```

Comment:
**(`-dd`, `-ddd`, etc)** - I even don't know how many `d`'s can be used (up to 8 or 9 probably).